src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1577,10 +1566,11 @@ internal static uint GetRefEscape(BoundExpression expr, uint scopeOfTheContainin
                 case BoundKind.ConditionalOperator:
                     var conditional = (BoundConditionalOperator)expr;
 
-                    // byref conditional defers to one operand. Since are here without errors, both operands must match.
                     if (conditional.IsByRef)
                     {
-                        return GetRefEscape(conditional.Consequence, scopeOfTheContainingExpression);
+                        // ref conditional defers to its operands
+                        return Math.Max(GetRefEscape(conditional.Consequence, scopeOfTheContainingExpression),
```

Comment:
Are we sure we want to relax the ref-safe-to-escape for ternary? Might that cause problems once we allow ref re-assignment (ie. the ternary could be used on the left)?

Reply:
It cannot be on the left of ref assignment.

Reply:
It would ned to be a ref to a ref.